p2p/peer_tracker.go

```diff
@@ -105,7 +143,19 @@ func (p *peerTracker) track() {
 	}
 }
 
-func (p *peerTracker) connected(pID peer.ID) {
+// getPeers returns the tracker's currently tracked peers.
+func (p *peerTracker) getPeers() []libpeer.ID {
+	p.peerLk.RLock()
+	defer p.peerLk.RUnlock()
+
+	peers := make([]libpeer.ID, 0, len(p.trackedPeers))
+	for peer := range p.trackedPeers {
+		peers = append(peers, peer)
+	}
+	return peers
```

Comment:
Considering that we take the first 4 for the request, we should sort them by the score and ensure the first 4 are the best ones

Reply:
Can make this into an issue.

Reply:
https://github.com/celestiaorg/go-header/issues/91